src/table/operations.jl

```diff
@@ -0,0 +1,60 @@
+import Base: filter, map, reduce
+
+"""
+    map(f, d::DTableRows)
```

Comment:
```suggestion
    map(f, d::DTableRows) -> DTable
```

Reply:
Doesn't return a DTable if that's the meaning